server/channels/store/storetest/settings.go

```diff
@@ -49,26 +48,12 @@ func log(message string) {
 	}
 }
 
-func getDefaultMysqlDSN() string {
```

Comment:
I think we should keep these.  There is no scenario where the const default DSN works in CI.